src/app/helptext/sharing/nfs/nfs.ts

```diff
@@ -22,6 +22,16 @@ export const helptext_sharing_nfs = {
   ),
   validators_path: [Validators.required],
 
+  placeholder_alias: T("Alias"),
+  tooltip_alias: T('To define a custom shortcut for the <i>Path</i>, \
+ enter a memorable name. Example: <i>/nfsshare</i>. NFS clients \
+ can connect using the <i>Alias</i> instead of the <i>Path</i> \
+ when the share is active. A share with multiple paths requires \
+ either not using aliases or defining an alias for each path in \
+ the share.'),
+  error_alias: T('The <i>Alias</i> field can either be left empty or \
```

Comment:
the <i> </i> tags on the error do not translate to actual italics for me (are these errors displayed through innerHTML?)

Reply:
![screenshot-20200729-113211](https://user-images.githubusercontent.com/1542251/88838980-3355a200-d18f-11ea-91fd-3fb78717800c.jpg)